exports.show = async (req, res) => {
    try {

    } catch (error) {

    }
}

exports.create = async (req, res) => {
    try {

    } catch (error) {
        return res.status(400).json({message:`Что то пошло не так, попробуйте снова`})
    }
}

exports.update = async (req, res) => {
    try {

    } catch (error) {

    }
}

exports.delete = async (req, res) => {
    try {

    } catch (error) {

    }
}

exports.delete = async (req, res) => {
    try {

    } catch (error) {

    }
}